fetch.bs

```diff
@@ -4903,8 +4951,60 @@ initially a new {{AbortSignal}} object.
  <var>init</var>])</code>
  <dd>Returns a new <var>request</var> whose {{Request/url}} property is <var>input</var> if
  <var>input</var> is a string, and <var>input</var>'s {{Request/url}} if <var>input</var> is a
- {{Request}} object. The optional <var>init</var> argument allows for setting attributes
- appearing in {{RequestInit}} via object members of the same name.
+ {{Request}} object.
+
+ <p>The optional <var>init</var> argument allows for setting properties appearing in {{RequestInit}}
+ via object members of the same name. These are the object members that can be used:</p>
+
+ <dl>
+  <dt>{{RequestInit/method}}
+  <dd>Set to a string to set <var>request</var>'s {{Request/method}}.
+
+  <dt>{{RequestInit/headers}}
+  <dd>Set to a {{Headers}} object or an array of two-item arrays to set <var>request</var>'s
+  {{Request/headers}}.
+
+  <dt>{{RequestInit/body}}
+  <dd>Set to a {{Body}} object or null to set <var>request</var>'s <a>body</a>.
```

Comment:
Not quite correct. It's actually a BodyInit. Either link to BodyInit, or maybe spell out what a BodyInit is.

Reply:
I opted for linking if that's ok. I figured that soon `BodyInit` will have its own domintro so what a developer might get linked too shouldn't be too tough to understand.